test/fixtures/app.js

```diff
@@ -8,4 +8,34 @@ var app = loopback();
 
 app.set('restApiRoot', '/rest-api-root');
 
+// this is a hack to give the angular SDK a model to export
+// TODO: create and inject a model the "proper" way
```

Comment:
Here is the proper way (I didn't run the code, it may not work OOTB):

```js
app.dataSource('db', { connector: 'memory' });

var TestModel = app.registry.createModel('TestModel', { name: 'string' });
app.model(TestModel, { dataSource: 'db' });
```